go.mod

```diff
@@ -37,6 +37,7 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.7.1
 	github.com/stretchr/testify v1.7.0
+	github.com/swaggo/swag v1.7.1
```

Comment:
this seems unexpected in this PR. Can you avoid the go.mod and go.sum changes?

Reply:
I believe these are the lines removed by you in a previous PR because if I run `go mod tidy` + `go mod vendor` this is the current state I get. 

Do you mind to try to install swag locally using `make install-swag`  and then try again?